eslint.config.mjs

```diff
@@ -1,27 +1,10 @@
 import globals from 'globals';
 import pluginJs from '@eslint/js';
 
-const baseConfig = {
-  extends: 'eslint:recommended',
-  parserOptions: {
-    ecmaVersion: 6
-  },
-  env: {
-    node: true,
-    browser: true,
-    es6: true
-  }
-};
```

Comment:
why was this removed?

Reply:
There is no need to manually write these configurations as their default values are the latest versions, and also this format was incorrect with the latest version